### PR TITLE
chore: splitting the public config from the VM

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

File: qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+ENABLE_CSRF_PROTECTION: true
+
+# fence (at the moment) attempts a migration on startup. setting this to false will disable that
+# WARNING: ONLY set to false if you do NOT want to automatically migrate your database.
+#          You should be careful about incompatible versions of your db schema with what
+#          fence expects. In other words, things could be broken if you update to a later
+#          fence that expects a schema your database isn't migrated to.
+# NOTE: We are working to improve the migration process in the near future
+ENABLE_DB_MIGRATION: true
```
Comment (paulineribeyre) [inline qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml]:
```suggestion
ENABLE_DB_MIGRATION: false
```
sorry, i misread your message earlier, we want all commons to have `ENABLE_DB_MIGRATION: false` going forward

File: qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+### DEPRECATED and replaced by OPENID_CONNECT + LOGIN_OPTIONS configs
+ENABLED_IDENTITY_PROVIDERS:
+  # ID for which of the providers to default to
+  default: 'google'
+  # Information for identity providers
+  providers:
+    google:
+      name: 'Google Login'
+    fence:
+      name: 'Fence Multi-Tenant Login'
```
Comment (paulineribeyre) [inline qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml]: let's convert this to LOGIN_OPTIONS please :) ask me or Di if you don't know how

File: qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  bucket1:
+    cred: 'CRED1'
+  bucket2:
+    cred: 'CRED2'
+    region: 'us-east-1' #optional but if specified avoids a call to GetBucketLocation which you may lack the AWS ACLs for.
+  bucket3:
+    cred: '*'
+  bucket4:
+    cred: 'CRED1'
+    role-arn: 'arn:aws:iam::role1'
```
Comment (paulineribeyre) [inline qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml]: let's make this `S3_BUCKETS: {}`

File: qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+
+# `DATA_UPLOAD_BUCKET` specifies an S3 bucket to which data files are uploaded,
+# using the `/data/upload` endpoint. This must be one of the first keys under
+# `S3_BUCKETS` (since these are the buckets fence has credentials for).
+DATA_UPLOAD_BUCKET: 'bucket1'
```
Comment (paulineribeyre) [inline qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml]:
```suggestion
DATA_UPLOAD_BUCKET: ''
```

File: qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+#       "Role Administrator"
+#
+#       NOTE2: It may be possible to further restrict the permissions in the future to
+#              be more fine-grained.
+#
+ENABLE_AUTOMATIC_BILLING_PERMISSION_SIGNED_URLS: false
+ENABLE_AUTOMATIC_BILLING_PERMISSION_SA_CREDS: false
+
+# whether or not to allow access to the /link/google endpoints
+ALLOW_GOOGLE_LINKING: true:
```
Comment (paulineribeyre) [inline qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml]:
```suggestion
ALLOW_GOOGLE_LINKING: true
```

File: qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml (hunk 2 above)
Comment (atharvar28) [reply qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml]: @paulineribeyre  do you want something like this 
```LOGIN_OPTIONS:
  - name: 'Login from Google'
    idp: google```

**Pull request merged**